MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>Be careful about feature selection. It would require one level more of train-validate-test split, or crossvalidation, since it is a hyperparameter search.</p><p>You might be overfitting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>